utils/aws_sigv4.cc

```diff
@@ -98,10 +101,12 @@ std::string get_signature(std::string_view access_key_id, std::string_view secre
 
     std::string payload_hash = apply_sha256(body_content);
     std::string canonical_request = fmt::format("{}\n{}\n{}\n{}\n{}\n{}", method, canonical_uri, query_string, canonical_headers.str(), signed_headers_str, payload_hash);
+    alog.debug("Canonical request: [{}]", canonical_request);
 
     std::string_view algorithm = "AWS4-HMAC-SHA256";
     std::string credential_scope = fmt::format("{}/{}/{}/aws4_request", datestamp, region, service);
     std::string string_to_sign = fmt::format("{}\n{}\n{}\n{}", algorithm, amz_date, credential_scope,  apply_sha256(canonical_request));
+    alog.debug("String to sign: [{}]", string_to_sign);
```

Comment:
Agree it looks somewhat random and not useful in general.

Reply:
Dropped